docs/file-formats/json-project/overview.md

```diff
@@ -46,6 +46,7 @@ A complete project definition looks like this:
     "enabled_licenses": [
         ...
     ],
+    "project_default_permissions": ...,
```

Comment:
does this mean you went for the approach with having both `project_default_permissions` and `permissions` in step 2?